hive/envs/gym_env.py

```diff
@@ -29,7 +33,19 @@ def create_env(self, env_name, **kwargs):
         Args:
             env_name (str): Name of the environment
         """
-        self._env = gym.make(env_name, **kwargs)
+        env = gym.make(env_name)
+
+        wrapper_config = kwargs.get("wrappers", None)
+        if isinstance(wrapper_config, list) and len(wrapper_config) > 0:
+            wrapper_config = {
+                "name": "CompositeEnvWrapper",
+                "kwargs": {"wrapper_list": wrapper_config},
+            }
+        if wrapper_config is not None:
+            wrapper_fn, _ = get_wrapper(wrapper_config, "wrappers")
+            env = wrapper_fn(env)
```

Comment:
This isn't super clean, CompositeEnvWrapper isn't really necessary. Something better would be add a parameter to the init called `env_wrappers: List[Wrapper]`. Then below:
```suggestion
def create_env(self, env_name, env_wrappers, **kwargs):
        self._env = gym.make(env_name)
        
        if env_wrappers is not None:
            self._env = apply_wrappers(env, env_wrappers)

def apply_wrappers(env, env_wrappers):
    for wrapper in env_wrappers:
        env = wrapper(env)
    return env         
```

`apply_wrappers` and the EnvWrapper definition should be moved to their own file. This way, the wrapper logic is also all gym independent.